### ISSUE TITLE
accessing styles using bracket notation counts as a failure

### ISSUE THREAD

**Issue by beeglebug** [Intellicode/eslint-plugin-react-native#182]
I would expect the following to pass, but it does not.

```javascript
const styles = StyleSheet.create({
  foo: { /* styles */ }
})

() => {
  const bar = 'foo'
  return <View style={styles[bar]} />
}
```

**Comment by Intellicode** [comment]
hi @beeglebug, in this case it might be possible to inline the key, but for more complex cases where `bar` is dynamic it will be hard to infer which styles are referenced. In these cases you might want to disable the rule for that specific line
